DataFormats/ForwardDetId/src/HGCSiliconDetIdToROC.cc

```diff
@@ -0,0 +1,64 @@
+#include "DataFormats/ForwardDetId/interface/HGCSiliconDetIdToROC.h"
+
+HGCSiliconDetIdToROC::HGCSiliconDetIdToROC() {
+  const int rocMax(6), cellRows(4), cellColumn(2);
+  const int type[rocMax]               = {0,0,1,1,2,2};
+  const int start1[rocMax][cellColumn] = {{3,2},{1,0},{1,2},{3,4},{7,6},{5,4}};
+  const int start2[rocMax]             = {0,0,0,0,4,4};
+
+  // key = ROC #; value = vector of trigger cell U.V
+  for (int roc=0; roc<rocMax; ++roc) {
+    std::vector<std::pair<int,int> > cells;
+    if (type[roc] == 1) {
+      for (int i1=0; i1<cellColumn; ++i1) {
+	int v = start2[roc];
+	int u = start1[roc][i1];
+	for (int i2=0; i2<cellRows; ++i2) {
+	  cells.emplace_back(std::make_pair(u,v));
```

Comment:
`make_pair` within `emplace_back` is redundant; just do `emplace_back(u,v)`
(in the two cases below as well)